src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigMapResource.java

```diff
@@ -29,6 +40,7 @@
 public class ConfigMapResource {
 
     private final ConfigService configService;
+    private final ConfigurationService configurationService;
```

Comment:
What is the difference between ConfigService and ConfigurationService?

Reply:
first written by @monastyrev, and use commit hash, and when it different update config, and used only for get
ConfigurationService it's low level service, it can update configuration directly 

naming confuse me too, but i just use it

Reply:
How could we improve this?

Reply:
ConfigService it's commons interface, and now too many project need this name
ConfigurationService it's best name for service than contains logic works with configuration

Do you have any suggestions?

Reply:
Could we extend ConfigService?